capa/features/extractors/dnfile/helpers.py

```diff
@@ -315,7 +317,6 @@ def is_dotnet_mixed_mode(pe: dnfile.dnPE) -> bool:
 
 
 def iter_dotnet_table(pe: dnfile.dnPE, name: str) -> Iterator[Any]:
```

Comment:
could we add some documentation and/or better type hint that describes the return value?

Reply:
update type checking - see most recent commits.